src/sentry/integrations/metric_alerts.py

```diff
@@ -80,6 +80,11 @@ def incident_attachment_info(incident, new_status: IncidentStatus, metric_value=
             },
         )
     )
+    if unfurl:
```

Comment:
Instead of redefining `title_link`, I think you should do: 
```py
if unfurl:
	# This URL is needed for the Slack unfurl, but nothing else.
	title_link = absolute_uri(        
		...
else:
    title_link = absolute_uri(
		...
```